src/plugins/data/public/query/query_string/dataset_service/lib/index_type.ts

```diff
@@ -72,7 +73,7 @@ export const indexTypeConfig: DatasetTypeConfig = {
     }
   },
 
-  fetchFields: async (dataset) => {
+  fetchFields: async (_, dataset: Dataset): Promise<DatasetField[]> => {
```

Comment:
NIt: Let's add the full name along with the _ for code readability

Reply:
Sure, added this in update commit [ee287e8](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8337/commits/ee287e8f87a614516ee49c2f7946145f3ef91092)